lang/python/2to3/files/2to3.py

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/python3
```

Comment:
You will need to replace this line with `$(STAGING_DIR_HOST)/bin/python3` (with the actual path for `$(STAGING_DIR_HOST)`) in `Host/Install`.

Reply:
okay, understand

Reply:
buttt it will work only on host purpose ?
If we need to do also a target/package with the same files it will not work I suppose...

Reply:
Same, okay with the first comment...